src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -689,7 +704,7 @@ public function merge(self $container)
     /**
      * Returns the configuration array for the given extension.
      *
-     * @return array An array of configuration
+     * @return array<array<string, mixed>> An array of configuration
```

Comment:
just `array<string, mixed>`, isn't it?

Reply:
loadFromExtension is doing 
```
$this->extensionConfigs[$namespace][] = $values;
```
where $values is `array<string, mixed>`
so `$this->extensionConfigs[$name]` is `array<array<string, mixed>>`.